scripts/auto-rebase/rebase.sh

```diff
@@ -532,6 +532,7 @@ update_manifests() {
     sed -i 's|REPLACE_COREDNS_IMAGE|{{ .ReleaseImage.coredns }}|' "${REPOROOT}"/assets/components/openshift-dns/dns/daemonset.yaml
     sed -i 's|REPLACE_RBAC_PROXY_IMAGE|{{ .ReleaseImage.kube_rbac_proxy }}|' "${REPOROOT}"/assets/components/openshift-dns/dns/daemonset.yaml
     sed -i 's|REPLACE_CLUSTER_IP|{{.ClusterIP}}|' "${REPOROOT}"/assets/components/openshift-dns/dns/service.yaml
+    sed -i 's|forward.*|forward . {{if eq .UpstreamResolver ""}}/etc/resolv.conf{{else}}{{.UpstreamResolver}}{{end}} {|' "${REPOROOT}"/assets/components/openshift-dns/dns/configmap.yaml
```

Comment:
The value in this expression doesn't match the value in the asset file at the top of the PR.

Reply:
Good catch!  I forgot to update the rebase script when fixing the comment.
Updated.